### DISCUSSION TITLE
Disable blue LED on Esp8266 Wemos D1 mini

### DISCUSSION THREAD

**Discussion by libtek** [awawa-dev/HyperSPI#25] [General]
Just wondering if there's a compile time option or something else to disable the solid blue LED on the Wemos D1 mini when flashing HyperSPI?  I don't currently have it in a case or on a proto/circuit board, so the continuous light can be distracting (even behind the TV).  Thanks!

**Comment by awawa-dev** [comment]
Hi
The onboard LED is probably connected to GPIO2 output. Unfortunately Esp8266 is missing internal muxer to use different than GPIO2 output for HyperSPI.
